Formula/libmf.rb

```diff
@@ -0,0 +1,41 @@
+class Libmf < Formula
+  desc "open source tool for approximating an incomplete matrix."
+  homepage "https://www.csie.ntu.edu.tw/~cjlin/libmf/"
+  url "http://www.csie.ntu.edu.tw/~cjlin/libmf/libmf-2.01.zip"
+  sha256 "75bb98a4e8f1a318d7d07556583727b4e301759904570bf527972d964d34ef30"
+  patch :DATA
+
+  def install
+    system "make"
+    system "make", "lib"
+    cp "./mf-predict", bin
+    cp "./mf-train", bin
+    cp "./libmf.so.2", lib
+  end
+
+  test do
+    system "mf-predict"
```

Comment:
Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Also please call executables with full paths in testing, i.e. `system bin/"mf-predict"`.